build.sbt

```diff
@@ -1,6 +1,19 @@
-name := "spark-web-model"
+lazy val root = project.in(file("."))
```

Comment:
This file is actually most important one. You can go without `.scala` files in `project` directory at all. But it is just very convenient to have real-Scala helpers instead of weird-looking-DSL.